packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -838,6 +838,9 @@ function use<T>(usable: Usable<T>): T {
   throw new Error('An unsupported type was passed to use(): ' + String(usable));
 }
 
+const MEMO_CACHE_UNSET_SENTINEL: symbol = Symbol.for(
+  'react.usememocache_sentinel',
+);
```

Comment:
I originally thought of putting this to <https://github.com/facebook/react/blob/main/packages/shared/ReactSymbols.js>